app/presenters/tasks/ioj.rb

```diff
@@ -8,29 +8,18 @@ def path
       end
     end
 
-    def not_applicable?
-      return true if crime_application.cifc?
-
-      false
-    end
-
     def can_start?
       fulfilled?(CaseDetails)
     end
 
     def in_progress?
-      crime_application.ioj_passport.any? || ioj.present?
+      ioj.present?
     end
 
-    def completed?
```

Comment:
I think removing this has meant that the tasklist badge doesn't show complete anymore